webview-ui/src/ClusterProperties/ClusterDisplayToolTip.tsx

```diff
@@ -24,7 +24,7 @@ export function ClusterDisplayToolTip(props: ClusterDisplayToolTipProps) {
                         <th>Preview</th>
                     </tr>
                     {props.clusterInfo.supportedVersions.map((v) => (
-                        <tr key={v.version}>
+                        <tr key={v.version} style={{ outline: "thin solid" }}>
```

Comment:
@Tatsinnit Only suggestion is to add this to css and add a class attribute to tr. just a suggestion, this also looks fine to me.